### PR TITLE
Add debugging statements for identifying slowness

### PR DESCRIPTION
**Pull request by jwodder**
Only for testing, not for merging.

### CODE REVIEW

File: tools/backups2datalad.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def main(
     logging.basicConfig(
         format="%(asctime)s [%(levelname)-8s] %(name)s %(message)s",
         datefmt="%Y-%m-%dT%H:%M:%S%z",
-        level=logging.INFO,
+        level=logging.DEBUG,
```
Comment (yarikoptic) [inline tools/backups2datalad.py:116]: let's just add use of e.g `BACKUP2DATALAD_LOG_LEVEL` env var which would provide numeric value to pass (or do it more properly with cli option etc), so we do not need to change the script for this and I will just merge and I will try it out to identify.  Will be useful in the long term anyways

Comment (jwodder) [reply tools/backups2datalad.py]: `--log-level` option added (Requires `click-loglevel` package).

Comment (yarikoptic) [reply tools/backups2datalad.py]: ha -- you made an entire package for that? heheh ;)
ok to out of draft so I could merge?

Comment (jwodder) [reply tools/backups2datalad.py]: Taken out of draft.

**Comment by yarikoptic** [comment]
despite the description, following the brief discussion I think it will be useful to have, thus merging.

**Pull request merged**